tools/pipelines/templates/build-npm-package.yml

```diff
@@ -438,81 +438,32 @@ stages:
-                workingDirectory: ${{ variables.toolAbsolutePath }}
-                # Note: $(ado-feeds-build) and $(ado-feeds-office) come from the ado-feeds variable group
-                script: |
-                  echo Initialize package
-                  npm init --yes
-
-                  echo Generating .npmrc
-                  echo "registry=https://registry.npmjs.org" >> ./.npmrc
-                  echo "always-auth=false" >> ./.npmrc
-
-                  echo "@fluidframework:registry=${{ variables.feed }}" >> ./.npmrc
-                  echo "@fluid-experimental:registry=${{ variables.feed }}" >> ./.npmrc
-
-                  # This is confusing, but we are using the .npmrc here to load dependencies from different feeds. These
-                  # scopes must be loaded from the "dev" feed because that is the only place they are published. Ideally
-                  # this logic will be centralized outside of CI in the future.
-                  echo "@fluid-internal:registry=${{ variables.devFeed }}" >> ./.npmrc
-                  echo "@fluid-private:registry=${{ variables.devFeed }}" >> ./.npmrc
-
-                  # This scope must be loaded from the "build" feed
-                  echo "@ff-internal:registry=$(ado-feeds-build)" >> ./.npmrc
-
-                  # This scope must be loaded from the internal Office feed
-                  echo "@microsoft:registry=$(ado-feeds-office)" >> ./.npmrc
-
-                  echo "always-auth=true" >> ./.npmrc
-                  cat .npmrc
-
-            - task: npmAuthenticate@0
-              displayName: 'npm authenticate (internal feed)'
-              inputs:
-                workingFile: ${{ variables.toolAbsolutePath }}/.npmrc
-
-            - task: Bash@3
-              displayName: 'List report.json and .npmrc'
+              displayName: 'List report.json'
               inputs:
                 targetType: 'inline'
                 workingDirectory:  ${{ parameters.buildDirectory }}
                 script: |
-                  echo "Build Directory is ${{ parameters.buildDirectory }}"
-                  ls -la '${{ parameters.buildDirectory }}/artifacts/bundleAnalysis/@fluid-example/bundle-size-tests';
-                  cat '${{ variables.toolAbsolutePath }}/.npmrc';
+                  echo "Build Directory is ${{ parameters.buildDirectory }}";
+                  BUNDLE_SIZE_TESTS_DIR="${{ parameters.buildDirectory }}/artifacts/bundleAnalysis/@fluid-example/bundle-size-tests";
+                  echo "Contents of $BUNDLE_SIZE_TESTS_DIR:";
+                  ls -la '$BUNDLE_SIZE_TESTS_DIR';
 
-            - task: Bash@3
-              displayName: 'Prepare telemetry-generator for bundle size'
-              inputs:
-                targetType: 'inline'
-                workingDirectory:  ${{ variables.toolAbsolutePath }}
-                script: |
-                  ls -la '${{ variables.toolAbsolutePath }}';
-                  ls -la '${{ parameters.buildDirectory }}';
```

Comment:
The template doesn't do this but it doesn't have much use anyway at this point (might have when initially troubleshooting this)